### ISSUE TITLE
Create an animated tour pattern

### ISSUE THREAD

**Issue by gdoyle1** [patternfly/patternfly-design#984]
**Low priority** - this should come after #914 

This issue directly relates to #914 . When thinking about tour patterns, there is almost always animation involved. OpenShift has applied this already in their latest release, adding animation to point out specific items on the UI to bring them to their attention.

The animation group has done some exploration of what this could look like as well when this type of pattern is applied to a navigation item or elsewhere. [Google drive folder](https://drive.google.com/drive/folders/1gviJq2jWr0kZHetDrC8KRpC_jgXJ59FV?usp=sharing) (see circular pulse, menu pulse 1, menu pulse, and rectangular pulse)

[OpenShift prototype](https://meet.google.com/linkredirect?authuser=0&dest=https%3A%2F%2Fxd.adobe.com%2Fview%2Fdd20d755-8def-4536-af2a-86ba83a8080c-8873%2F)

**Comment by mcarrano** [comment]
No longer relevant
